tests/unit/setup.js

```diff
@@ -2,6 +2,8 @@ import vueSnapshotSerializer from './serializer.js';
 
 expect.addSnapshotSerializer(vueSnapshotSerializer);
 
+global.document = global.window.document;
+document.body.innerHTML = '<div id="app"></div>';
```

Comment:
Now that `main.js` is being tested we need to mock out `#app` so it can mount to something without throwing an error.